recipes/ua-nodeset/all/conanfile.py

```diff
@@ -10,34 +12,36 @@ class UaNodeSetConan(ConanFile):
     description = "UANodeSets and other normative files which are released with a specification"
     homepage = "https://github.com/OPCFoundation/UA-Nodeset"
     url = "https://github.com/conan-io/conan-center-index"
-    topics = ("opc-ua-specifications", "uanodeset", "normative-files", "companion-specification")
-
-    no_copy_source = True
+    topics = ("opc-ua-specifications", "uanodeset", "normative-files", "companion-specification", "pre-built")
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    package_type = "application"
```

Comment:
```suggestion
    package_type = "build-scripts"
```
The package does not contain a single executable file, but only consumable files .xml, .csv, .bsd, .ua ...

Reply:
Thanks! I hope this fixes the linter issue.